src/exportImage.py

```diff
@@ -2,28 +2,64 @@
 from math import floor
 from log.logger import Logger
 from src.config import read_config
+import requests
+from datetime import datetime
+import json
 
 
 class ExportImage:
     def __init__(self):
         self.logger = Logger()
         self.__config = read_config()
 
-    def export(self, offset, bbox, filename, url):
+    def export(self, offset, bbox, filename, url, taskid):
         try:
+            es_obj = { "taskId": taskid, "filename": filename, "link": "/test/link"}
```

Comment:
change "link" to URI (full path to created file)

Reply:
fixed in cb74980